### PR TITLE
Unixdaddy sheffield

### PR DESCRIPTION
**Pull request by unixdaddy**
This pull request adds a file about where to eat in sheffield / UK ;-)

### CODE REVIEW

File: unixdaddy-sheffield.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Sheffield, UK
+
+## Places to Eat
+- Mumtaz for Indian food
+- Macdonalds for stomach pain ;-)
+- Mum's for GOOD food
```
Comment (jrglez) [inline unixdaddy-sheffield.md:6]: What kind of food is served here?

File: unixdaddy-sheffield.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+## Places to Visit
```
Comment (jrglez) [inline unixdaddy-sheffield.md:8]: Any interesting museums?

**Review by jrglez — CHANGES_REQUESTED** [review]
Could you add some more info?

File: unixdaddy-sheffield.md (hunk 1 above)
Comment (unixdaddy) [reply unixdaddy-sheffield.md:6]: added some wonderful places

**Review by unixdaddy — COMMENTED** [review]
looks good

**Pull request merged**